met-web/src/components/engagement/form/AddSurveyBlockTabs.tsx

```diff
@@ -25,13 +58,19 @@ const AddSurveyBlockTabs = () => {
                     </MetTabList>
                 </Box>
                 <MetTabPanel value={SubmissionStatus[SubmissionStatus.Upcoming]}>
-                    <RichTextEditor initialHTMLText={upcomingText} />
+                    <RichTextEditor
+                        handleEditorStateChange={handleRichDescriptionChange}
+                        initialHTMLText={upcomingText}
+                    />
                 </MetTabPanel>
                 <MetTabPanel value={SubmissionStatus[SubmissionStatus.Open]}>
-                    <RichTextEditor initialHTMLText={openText} />
+                    <RichTextEditor handleEditorStateChange={handleRichDescriptionChange} initialHTMLText={openText} />
```

Comment:
handleRichDescriptionChange? The description field is another field in engagement form, this doesn't change that right?